client/src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect, useState } from 'react';
 import GeoJSONMap from './components/geoJSONMap';
-import FileUploadForm from './components/uploadForm'
+// import FileUploadForm from './components/uploadForm'
```

Comment:
any reason for cutting out the forms? the upload form and system can stay as it is.